content/departments/engineering/teams/graph/index.md

```diff
@@ -0,0 +1,17 @@
+# Graph Team
```

Comment:
This can go in a different PR, but this should probably integrate the content from the old code intel page and then that page can be obsoleted

https://handbook.sourcegraph.com/departments/engineering/teams/code-intelligence/

Reply:
I started this but there is a TON of content there and I pulled over just what seemed most pertinent. We will need to do a future pass also.